CHANGES.rst

```diff
@@ -79,6 +79,8 @@ Bug Fixes
 
 - Fixes viewer toolbar items losing ability to bring up right-click menu. [#2605]
 
+- Disable closing tab-groups of viewers to prevent closing required viewers. [#2625]
+
```

Comment:
```suggestion
```

Reply:
I suppose that's true!  Any config that had the tabs enabled at all had this manually hidden already.